doc/source/wsrep-system-index.rst

```diff
@@ -418,6 +418,64 @@ Possible values for this variable are:
 .. |binlog_format| replace:: ``binlog_format``
 .. _binlog_format: https://dev.mysql.com/doc/refman/8.0/en/binary-log-setting.html
 
+.. variable:: wsrep_ignore_apply_errors
+
+    :cli: ``--wsrep-ignore-apply-errors``
+    :conf: Yes
+    :scope: Global
+    :dyn: Yes
+    :default: 0
+    
+Defines the rules of wsrep applier behavior on errors. You can change the settings by editing the ``my.cnf`` file under ``[mysqld]`` or at runtime.
+
+.. note::
+
+    In |PXC| version 8.0.19-10, the default value has changed from ``7`` to ``0``. If you have been working with an earlier version of the PXC 8.0 series, you may see different behavior when upgrading to this version or later.
+
+The variable has the following options:
+
+.. list-table::
+    :widths: 25 30
+    :header-rows: 1
+    
+    * - Value
+      - Description
+    * - WSREP_IGNORE_ERRORS_NONE
+      - All replication errors are treated as errors and will shutdown the node or MySQL server (default behavior)
```

Comment:
node or MySQL server is the same thing here. We should choose one.

Reply:
Fixed.